sympy/integrals/integrals.py

```diff
@@ -758,9 +763,19 @@ def _eval_integral(self, f, x, meijerg=None, risch=None, manual=None,
             else:
                 if i:
                     # There was a nonelementary integral. Try integrating it.
-                    return result + i.doit(risch=False)
+                    a="ans"
```

Comment:
@jksuom Yeah, actually, I don't know why the integrals.py file got pushed to the commit here. I pushed those files for a PR at a different branch. 

For this issue, I created a new branch, and pushed the PR with only these files changed - expr.py and perm_groups.py , but then somehow the other 2 files, which were part of my previous commit, also came into this branch. I don't know how. :( 

Sorry for the same. Do I need to change it? Or is it fine? 

Thanks! :smile: